lib/taxes/taxes-rules-fr.ts

```diff
@@ -435,8 +443,13 @@ const getFrTaxableEventFromGainsAndLossEvent = (
       total: (sellPriceEur - acquisitionValueEur) * event.quantity,
     },
     acquisitionGain: {
-      perShare: acquisitionValueEur - acquisitionCostEur,
-      total: (acquisitionValueEur - acquisitionCostEur) * event.quantity,
+      perShare:
+        (acquisitionValueEur - acquisitionCostEur) * event.fractionFrIncome,
```

Comment:
Hmm, applying it to the value per share gives us more "Details" in the print view

Reply:
I think it is confusing because it requires more thinking when 
```
acquisition cost: 10€
acquisition value: 20€
nb shares: 10
acquisition gain per share: 10€

percent applicable for french taxes: 75%
total computed acquisition gain: 75€
```

Versus

```
acquisition cost: 10€
acquisition value: 20€
nb shares: 10
percent applicable for french taxes: 75%

acquisition gain per share: 7.5€
total computed acquisition gain: 75€
```

I find the former easier to follow, but it might just be me.

BTW, maybe the % line can be hidden if everything is 100%, WDYT?

Reply:
I personally like the latter better but I won't die on that hill. Can we meet in the middle and hide the % line if it's 100%?

Reply:
sure!